### PR TITLE
Wake-on-Lan

### PR DESCRIPTION
**Pull request by FernetMenta**
This is something vnsi has had for a long time.

### CODE REVIEW

**Review by ksooo — COMMENTED** [review]
Nice feature. Thanks for that.

* As this is release containing a new feature, please increase the minor version number, not the micro. => 5.6.0
* Please update changelog.txt

File: pvr.hts/resources/settings.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <setting id="user" type="text" label="30004" default="" />
     <setting id="pass" type="text" label="30005" option="hidden" enable="!eq(-1,)" default="" />
     <setting id="connect_timeout" type="slider" option="int" range="1,60" label="30006" default="10" />
-    <setting id="response_timeout" type="slider" option="int" range="1,60" label="30007" default="5" />
```
Comment (Jalle19) [inline pvr.hts/resources/settings.xml:11]: This seems to have been accidentally removed

**Review by Jalle19 — CHANGES_REQUESTED** [review]

File: pvr.hts/resources/settings.xml (hunk 1 above)
Comment (FernetMenta) [reply pvr.hts/resources/settings.xml:11]: oops, thx

**Comment by FernetMenta** [comment]
updated

**Review by ksooo — APPROVED** [review]
Thank you.

**Pull request merged**